.travis.yml

```diff
@@ -2,31 +2,55 @@ dist: xenial
 sudo: false
 language: scala
 
-scala:
-  - 2.12.8
-  - 2.11.12
+scala_version_211: &scala_version_211 2.11.12
+scala_version_212: &scala_version_212 2.12.8
+scala_version_213: &scala_version_213 2.13.0-M5
+scala_version_docs: &scala_version_docs 2.12.8
```

Comment:
I thought this could be:

```yaml
scala_version_docs: &scala_version_docs *scala_version_212
```

But that's not true.  Can a Certified Yaml Engineer please advise?